cmd/solana_exporter/exporter_test.go

```diff
@@ -128,22 +130,22 @@ func (c *staticRPCClient) GetVersion(ctx context.Context) (string, error) {
 }
 
 //goland:noinspection GoUnusedParameter
-func (c *staticRPCClient) GetVoteAccounts(
-	ctx context.Context,
-	params []interface{},
-) (*rpc.VoteAccounts, error) {
+func (c *staticRPCClient) GetVoteAccounts(ctx context.Context, params []interface{}) (*rpc.VoteAccounts, error) {
 	return &staticVoteAccounts, nil
 }
 
 //goland:noinspection GoUnusedParameter
 func (c *staticRPCClient) GetBlockProduction(
-	ctx context.Context,
-	firstSlot *int64,
-	lastSlot *int64,
+	ctx context.Context, firstSlot *int64, lastSlot *int64,
 ) (*rpc.BlockProduction, error) {
 	return &staticBlockProduction, nil
 }
 
+//goland:noinspection GoUnusedParameter
+func (c *staticRPCClient) GetBalance(ctx context.Context, address string) (float64, error) {
+	return balances[address], nil
+}
```

Comment:
If you used named returns, and just returned the named return here, you'd not need the linter config.

Reply:
I would still get a warning that `ctx` is unused, or am I not understanding correctly?